app/src/main/java/com/asfoundation/wallet/wallet_validation/generic/WalletValidationActivity.kt

```diff
@@ -47,6 +56,17 @@ class WalletValidationActivity : BaseActivity(),
     validation_progress_animation.setMinAndMaxFrame(minFrame, maxFrame)
     validation_progress_animation.repeatCount = loopAnimation
     validation_progress_animation.playAnimation()
+    setupToolbar()
+  }
+
+  private fun setupToolbar() {
+    intent.getStringExtra(TOOLBAR_TITLE)
```

Comment:
I think this can be done e a more elegant way:

`  val appPackage: String by lazy {
    if (arguments!!.containsKey(PARAM_APP_PACKAGE)) {
      arguments!!.getString(PARAM_APP_PACKAGE)
    } else {
      throw IllegalArgumentException("application package name data not found")
    }
  }`